apps/client/src/utils/index.js

```diff
@@ -2,6 +2,11 @@ import isBoolean from "lodash.isboolean";
 import isNumber from "lodash.isnumber";
 import isString from "lodash.isstring";
 
+import { topics } from "../config";
+
+// Find a topic by the slug
+export const findTopicBySlug = (slug) => topics.find((t) => t.slug === slug);
```

Comment:
aren't we using this in the withTracking and withTopic too?

Reply:
For now I will refactor that too, but that all is going to be refactored soon.